plantisc2/tasks/db.yml

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
yes, that would be a good suggestion. But let's keep the code in the project and not use submodules for now.